client/components/GraphTest.js

```diff
@@ -42,10 +42,11 @@ export default class GraphTest extends Component {
   }
 
   render() {
+    console.log('DATA===', this.state.complaints)
     return (
       <div>
         <BarGraphTest rawData={this.state.complaints} />
-        <PieChart data={this.state.complaints} />
+        <PieChart rowData={this.state.complaints} />
```

Comment:
Just for future reference, the PieChart will be expecting 'rowData' in its props.